src/main/java/edu/cornell/kfs/module/purap/rest/jsonObjects/PaymentRequestDto.java

```diff
@@ -0,0 +1,191 @@
+package edu.cornell.kfs.module.purap.rest.jsonObjects;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.core.api.util.type.KualiDecimal;
+
+public class PaymentRequestDto {
```

Comment:
Are there places in this DTO and related DTO Classes where using final makes sense?

Reply:
I can change these variables to final, but that might get pretty involved.  Do you think switching to final on the variables is critical?